Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+EXPOSE 5005:5005
+
+RUN apt-get update
+RUN apt-get install -y build-essential libxml2-dev python3-dev python3-pip zlib1g-dev
```

Comment:
I believe best practice is to combine these and remove the cache at the end:
```dockerfile
RUN apt-get update && \
    apt-get install -y ... && \
    rm -rf /var/lib/apt/lists/*
```
see https://docs.docker.com/v17.12/develop/develop-images/dockerfile_best-practices/#run

Reply:
Good catch, I had this in to debug, but forgot to re-"&& \"-ify it. WIll fix.